### ISSUE TITLE
After clicking "Add" on a field with multiple options (radio, select, checkbox), I shouldn't have to click in the field to start typing

### ISSUE THREAD

**Issue by jshorland** [ushahidi/platform#2073]
### Expected behaviour
After clicking "Add" on a field with multiple options (radio, select, checkbox), it should automatically place my cursor in the field and I should be able to just start typing

### Actual behaviour
Instead, I have to manually click in the option field to start typing. This kind of thing really adds up on time to set up my surveys, as well as general annoyance with the platform.

### Steps to reproduce the behaviour/error

Aha! Link: https://ushahiditeam.aha.io/features/PROD-521

**Comment by rowasc** [comment]
Confirmed this still happens

**Comment by eriyanto87** [comment]
@Angamanga Hi Anna. I'm an Outreachy internship applicant. Can I get assigned to this issue?

**Comment by OluyemisiA** [comment]
@Angamanga Hello Anna. I'm an Outreachy internship applicant, can I get assigned to this issues?
Thank you

**Comment by rowasc** [comment]
Hey @eriyanto87 please go ahead, I'm assigning it to you :) Thanks!

@OluyemisiA please check https://github.com/ushahidi/platform/issues?q=is%3Aopen+is%3Aissue+label%3A%22Community+Task%22+milestone%3A%22Hacktoberfest+%22 for unclaimed tickets . Thanks!

**Comment by eriyanto87** [comment]
@rowasc Thanks! I will start working on it as soon as I can!

**Comment by rowasc** [comment]
Brilliant, thank you !

**Comment by eriyanto87** [comment]
@rowasc I'm trying to go to the website https://ushahiditeam.aha.io/features/PROD-521 to see what the issue is but I am having trouble to make an account.

**Comment by Angamanga** [comment]
@eriyanto87 You don't need to care about that link, its for internal use only :). Sorry for the confusion!

**Comment by eriyanto87** [comment]
@Angamanga @rowasc Hello, I'm sorry I couldn't find where the issue is coming from.

**Comment by renujain31** [comment]
Hey @eriyanto87, so when you create a new survey, it says that the cursor should automatically point at Survey name and you should be able to start typing. But it doesn't. You have to set the focus on that input element every time you open up that page. 

I found that this issue is not just limited to survey, it is almost in the whole platform:
- while adding a new post to any survey
- while adding tasks in a survey
- while adding fields like checkboxes etc in a survey

Maybe we should come up with a more general way to make the first input element focusable @Angamanga @rowasc?
Let me know if you face any difficulty and I would be happy to help you.

**Comment by eriyanto87** [comment]
@renujain31 yes please! this is the first time I am looking at an open source code. I am grateful you are willing to help and explain. I'm not even sure will this code be?

**Comment by renujain31** [comment]
Okay, so a way to make elements focusable might just be to add `autofocus` as an attribute where the input is taken.

To find where the code is: (Hoping you have cloned and setup the platform-client repository)
- Inspect the webpage where you find that the element is not getting focused.
- Try to search for similar keyword present in that element in the editor you are editing the code in.

For example, in case of creating a post for any survey,
You would have to set the autofocus for the input box with placeholder: "Search for addresses, place names, or coordinates...". While inspecting that you can see that it uses a function "handleActiveSearch($event)". Woah! we found out what we need to search. Now you can search for that in the editor 
You would find that this is present [here](https://github.com/ushahidi/platform-client/blob/develop/app/main/posts/modify/location.html#L13). Now, and add `autofocus` in that element and test your changes.

PROTIP: Never search for an english phrase, there's a high probability that you may end up checking for 3 or more files. As we use translators and so what you see in inspect is generated by translators and therefore may not point you to the exact code. For example, here we are searching for "handleActiveSearch($event)" instead of "Search for addresses, place names, or coordinates..."

Hope this helps @eriyanto87 :) 

Ping me in case you need any other help. Hoping to see your first PR soon! 🥂

**Comment by renujain31** [comment]
Hey @eriyanto87 are you working on this? If not, can I take this up @rowasc?

**Comment by renujain31** [comment]
I have added a Pull request for it's fix. Can you please review it @rowasc. Thanks!
